README.md

```diff
@@ -31,6 +31,10 @@ Get started with $20 of free credits to experience the power of Gemini 2.5 Pro,
 - [Substack](https://blog.kilocode.ai/) (blog)
 - [Product Hunt](https://www.producthunt.com/products/kilocode) (vote)
 
+<p align="center">
+  <img src="https://raw.githubusercontent.com/Kilo-Org/kilocode/refs/heads/main/kilo.gif" width="100%" />
```

Comment:
Not sure width 100% is good here. I'd say ship it and experiment in different devices

Reply:
Just basing it off what Cline does: https://github.com/cline/cline/blob/main/README.md?plain=1#L8